### PR TITLE
Allow to force build type (gradle/ant) in buildozer.spec

### PR DESCRIPTION
**Pull request by maho**
I had situation when I had to force to NOT USE gradle. It was false positive detected, and in effect - build failed.

### CODE REVIEW

**Comment by inclement** [comment]
Thanks for the contribution. Would you mind calling the buildozer.spec option something like `android.java_build_tool` to match the p4a option, and placing the setting in the default.spec with the other `android.___` settings?

Also, did you make a p4a issue about this issue? We should be able to improve the autodetection if it's wrong sometimes (it's really just a best guess as it stands).

**Comment by metamarcdw** [comment]
This may also serve as a workaround for my issue #602. Hope to see it merged soon!

**Comment by mahomahomaho** [comment]
@inclement: I don't mind renaming the option. Just pushed the change

Re p4a issue - I don't think it's p4a issue. I rather find it as buildozer issue - wrongly detected tool used in p4a by existence (or not) of some files. 

Also, it's very hard to reproduce the error :). When I clean ~/.buildozer ./.buildozer, then sometimes p4a uses gradle, sometimes p4a uses ant, I don't see deterministic pattern in it.

File: buildozer/default.spec [hunk 1]
```diff
@@ -146,6 +146,9 @@ fullscreen = 0
 # bootstrap)
 #android.gradle_dependencies =
 
+# (str) build type ('auto' - default/auto detect, 'ant' - use ant, 'gradle' - use gradle)
+#android.java_build_tool = 'auto'
```
Comment (inclement) [inline buildozer/default.spec]: Am I right in thinking that `auto` should not actually have the `'` around it here?

**Comment by inclement** [comment]
Looks good - just one question annotated before merging.

Thanks!

**Comment by mahomahomaho** [comment]
could someone merge it to trunk?

**Comment by tito** [comment]
This PR is now outdated, and sdl2 bootstrap handle build type correctly.

**Pull request closed without merging**